contrib/kubernetes/deployment.yaml

```diff
@@ -23,7 +23,7 @@ spec:
             - containerPort: 6343
             - containerPort: 2055
             - containerPort: 2056
-          imagePullPolicy: Always
+          imagePullPolicy: Never
```

Comment:
This should be set for `Never` only for local-deployment.

Reply:
@ronensc maybe "IfNotPresent" fits both use-cases ... and actually if the code for OCP like in here https://cookbook.openshift.org/image-registry-and-image-streams/how-do-i-import-an-image-from-an-external-image.html works than you can keep the `Never` approach and add the code for OCP deployment to also load an image ... I really prefer the second, but it does require a bit of more work to check that this works also for OCP.

Reply:
I tried loading the FLP image into an OCP cluster and then started a pod with that image
```
$ oc import-image quay.io/netobserv/flowlogs-pipeline:latest --confirm
$ oc run  mypod --image=quay.io/netobserv/flowlogs-pipeline:latest --image-pull-policy=Never
```

But the pod couldn't find the image
```
$ oc get po
NAME    READY   STATUS              RESTARTS   AGE
mypod   0/1     ErrImageNeverPull   0          4s
```


Also, AFAIU, image streams are an OpenShift feature and there is no `import-image` equivalent in `kubectl`.
Do we want to introduce a new `oc` command to our Makefile?

Reply:
@ronensc I was looking on https://medium.com/@adilsonbna/importing-an-external-docker-image-into-red-hat-openshift-repository-c25894cd3199 

This seems to solve this ... but this is also not trivial and makes things even a bit more complicated 

So, what I suggest is a keep things as is and move to `IfNotPresent` mode ... this should work for both.
Also, small remark in the makefile of OCP that states that this uses the latest version from `quay.io` 

Hope this makes sense to you

We might improve in one of the future PR's

Reply:
@jotak @mariomac @jpinsonneau @OlivierCazade if any of you guys know how to push an image from local docker into OCP (above links are almost working, but not really :-) ))) we can use that to improve the process of ocp-deploy